### PR TITLE
[ENH] add anisotropic kurtosis, make dki default?

### PR DESCRIPTION
**Pull request by 36000**
This adds a new anisotropic kurtosis parameter. 

It also makes DKI tand fa/md/kfa/mk the default scalars for multishell data, and DTI FA/MD the default for single shell (our current default)

### CODE REVIEW

**Comment by 36000** [comment]
image

DKI KFA (left) vs DKI FA (right) on HCP subejct

**Comment by 36000** [comment]
@arokem  this is ready for review / merge!

File: AFQ/tasks/data.py [hunk 1]
```diff
@@ -773,6 +772,17 @@ def dki_mk(dki_tf):
     return dki_tf.mk()
 
 
+@pimms.calc("dki_kfa")
+@as_file('_odfmodel-DKI_desc-KFA_dwi.nii.gz')
+@as_fit_deriv('DKI')
+def dki_kfa(dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI mean kurtosis file
```
Comment (arokem) [inline AFQ/tasks/data.py:781]:
```suggestion
    the DKI kurtosis FA file
```

**Review by arokem — COMMENTED** [review]
Thanks for adding KFA! We might want to add a link to this paper somewhere in the documentation: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7028548/, which will help users understand how this differs from the DTI FA. 

Regarding making DKI the default, I would be fine with that, but I'd suggest adding a check that the data can be used for DKI (i.e., that it has multiple non-zero b-values) before starting the run, and raising an informative error when that's not the case, telling users how they should change their call (i.e., where to change into "dti_fa" and "dti_md").

**Pull request merged**